### DISCUSSION TITLE
Disable pagination controls but keep the pagination control with data when row selection is made

### DISCUSSION THREAD

**Discussion by beingmeak05** [KevinVandy/material-react-table#233] [Q&A]
Hi Kevin,

First of all, your material-react-table is just awesome!

I would like to disable pagination control when there is a row selection or page selection is made. Right now, when I try to programmatically disable the pagination, the control tries to list down all the rows, instead, I need it to display (example 10 rows) the rows list as it is and disable just the pagination controls until row selection remains true.

**Comment by KevinVandy** [comment]
You can disable just the pagination buttons using props in the `muiTablePaginationProps` prop. Full list of options are listed in MUI docs https://mui.com/material-ui/api/table-pagination/#props

Explained briefly in the MRT props here: https://www.material-react-table.com/docs/guides/pagination#customize-pagination-components

You might want something like this?

```jsx
muiTablePaginationProps={{
  backIconButtonProps: {
    disabled: isSomeRowSelected,
  },
  nextIconButtonProps: {
    disabled: isSomeRowSelected,
  },
}}
```

**Reply by beingmeak05** [reply]
Awesome! Thank you :)
